src/test/java/com/solvd/laba/carina/demo/JacketsPageTest.java

```diff
@@ -0,0 +1,125 @@
+package com.solvd.laba.carina.demo;
+
+import com.solvd.laba.carina.homework.pages.softwaretestingboard.MenJacketsPage;
+import com.solvd.laba.carina.homework.pages.softwaretestingboard.ProductToolbar;
+import com.zebrunner.agent.core.annotation.TestLabel;
+import com.zebrunner.carina.core.IAbstractTest;
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+import com.zebrunner.carina.core.registrar.tag.Priority;
+import com.zebrunner.carina.core.registrar.tag.TestPriority;
+import org.openqa.selenium.JavascriptExecutor;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.math.BigDecimal;
+import java.time.Duration;
+import java.util.Comparator;
+import java.util.stream.Collectors;
+
+public class JacketsPageTest implements IAbstractTest {
+
+    @Test
+    @MethodOwner(owner="ivan")
+    @TestPriority(Priority.P0)
+    @TestLabel(name="login", value={"successful login"})
+
+    public void testAddToCart(){
+        MenJacketsPage page = new MenJacketsPage(getDriver());
+        page.open();
+        page.assertPageOpened();
+
+        SoftAssert softAssert = new SoftAssert();
+
+        for(int i = 0; i < 3; i++){
+            page.addToCart(i);
+            page.waitForJSToLoad();
+            softAssert.assertTrue(page.checkIfInCart(i), String.format("The item number %d is not in the cart.", i));
+        }
+
+        softAssert.assertAll();
+    }
+
+    @Test
+    @MethodOwner(owner="Ivan")
+    @TestPriority(Priority.P0)
+    public void removeProductsFromCart(){
```

Comment:
The name of the test method must contain the word Test at the end - removeProductsFromCartTest